ML/Models/ScoreDetail.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatbee.ML.Models
+{
+    public class ScoreDetail : Input
+    {
+        public float score { get; set; }
```

Comment:
Proper case

Reply:
This was left over from previous projects, updated.